cache-web-app-arm-with-redis-cache-provision/readme.md

```diff
@@ -0,0 +1,4 @@
+# Create a Web App plus Redis Cache using a template
```

Comment:
For this template a 201 name would be good. Something like 201-web-app-with-redis-cache